src/everything.c

```diff
@@ -1 +1,6 @@
 #include "./everything.h"
+
```

Comment:
We can just add `--no-entry` to the compilation and that prevents the need to have a `main()`

Reply:
I added it for testing, forgot to remove